src/Sulu/Bundle/TranslateBundle/Tests/Functional/Translate/ExportTest.php

```diff
@@ -114,7 +115,13 @@ public function testExportWithFrontEnd()
 
         $expectedHash = file_get_contents(self::$fixturePath . '/shouldbes/sulu-test.frontend.en.xlf');
         $actualHash = file_get_contents(self::$fixturePath . '/sulu-test.frontend.en.xlf');
+        $actualHash = $this->removeTranslationIds($actualHash);
 
         $this->assertSame($expectedHash, $actualHash);
     }
+
+    private function removeTranslationIds($hash)
+    {
+        return preg_replace('/ id="(\w+)" /', ' ', $hash);
```

Comment:
This is necessary, because the `id` changed in Symfony 3.4, therefore we can't check the content of the entire file anymore, and be compatible to both versions. That's why I just remove the `id` here (since it doesn't really matter anyway).